integration-tests/tests/src/tests/schema.ts

```diff
@@ -17,34 +17,47 @@
 ////////////////////////////////////////////////////////////////////////////
 
 import { expect } from "chai";
-import { Realm } from "realm";
+import { openRealmBefore } from "../hooks";
+
+interface Test {
+  primary?: number;
+  value?: number;
+}
 
 describe("Realm schema", () => {
   describe("Default property values", () => {
-    it("can take a function as a default property value", () => {
-      interface Test {
-        dynamic?: number;
-      }
-
-      const realm = new Realm({
-        schema: [
-          {
-            name: "Test",
-            properties: {
-              dynamic: {
-                type: "int",
-                default: () => 42,
-              },
+    openRealmBefore({
+      schema: [
+        {
+          name: "Test",
+          primaryKey: "primary",
+          properties: {
+            primary: {
+              type: "int",
+              default: () => 42,
+            },
+            value: {
+              type: "int",
+              default: () => 13,
```

Comment:
We should test simple value-defaults as well, both for PK and non-PK (since they unfortunately have separate code paths). Is that covered by other tests?

Reply:
It is covered now